Source/WebInspectorUI/UserInterface/Models/LayoutTimelineRecord.js

```diff
@@ -37,6 +37,8 @@ WI.LayoutTimelineRecord = class LayoutTimelineRecord extends WI.TimelineRecord
 
         this._eventType = eventType;
         this._quad = quad || null;
+        this._area = area || null;
```

Comment:
is there really no way of getting a `quad` for LCP?  without this it wont be possible to highlight the layout area when the developer hovers over the record in the Layout timeline view :(

since we are providing a `domNode` we could have it at least highlight that instead?  id add something like this to `WI.LayoutTimelineView.prototype._showHighlightForRecord`
```js

```